### PR TITLE
[PyTorch][Edge] Tests for QuantizationFx API on lite modules

### PR DESCRIPTION
**Pull request by pavithranrao**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#60476**

# Context
Add tests for Lite modules that are quantized using fx API

Differential Revision: [D29306402](https://our.internmc.facebook.com/intern/diff/D29306402/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D29306402/)!

### CODE REVIEW

File: test/mobile/test_quantize_fx_lite_script_module.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                x = self.conv2(x)
+                return x
+
+        m = M().eval()
+        qconfig_dict = {"object_type": [(torch.nn.Conv2d, default_qconfig)]}
+        m = prepare_fx(m, qconfig_dict)
+        data = torch.randn(1, 1, 1, 1)
+        m = convert_fx(m)
+        # first conv is quantized, second conv is not quantized
+        node_list = [
```
Comment (iseeyuan) [inline test/mobile/test_quantize_fx_lite_script_module.py:67]: Looks like it's not used?

File: test/mobile/test_quantize_fx_lite_script_module.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                self.conv1 = nn.Conv2d(1, 1, 1)
+                self.conv2 = nn.Conv2d(1, 1, 1)
+
+            def forward(self, x):
+                x = self.conv1(x)
+                x = self.conv2(x)
+                return x
+
+        m = M().eval()
+        qconfig_dict = {"": default_qconfig, "module_name": [("conv1", None)]}
```
Comment (iseeyuan) [inline test/mobile/test_quantize_fx_lite_script_module.py:65]: Thanks for this update!

**Review by iseeyuan — APPROVED** [review]
LGTM! Would be great if @jerryzh168 and @supriyar could take a second look.

File: test/mobile/test_quantize_fx_lite_script_module.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    convert_fx,
+)
+from torch.testing._internal.common_quantization import NodeSpec as ns
+from torch.testing._internal.common_quantization import (
+    QuantizationLiteTestCase,
+    LinearModelWithSubmodule,
+)
+
+
+class TestFuseFx(QuantizationLiteTestCase):
```
Comment (supriyar) [inline test/mobile/test_quantize_fx_lite_script_module.py:22]: Why is this class called TestFuseFx?

**Review by supriyar — APPROVED** [review]
looks good!

**Pull request closed without merging**